source/_integrations/signal_messenger.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: "Signal Messenger"
+description: "Instructions on how to integrate Signal Messenger within Home Assistant."
+logo: signal_messenger.png
+ha_category:
+  - Notifications
+ha_release: 0.104
+---
+
+The `signal_messenger` integration uses the [Signal Messenger REST API](https://github.com/bbernhard/signal-cli-rest-api) to deliver notifications from Home Assistant to your Android or iOs device.
+
+## Setup
+ 
+The requirements are:
+
+- You need to set up the Signal Messenger REST API. 
+- You need a spare phone number to register with the Signal Messenger service. 
+
+
+Please follow those [instructions](https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/HOMEASSISTANT.md), to set up the Signal Messenger REST API. 
```

Comment:
content of this readme should be copied to HA repo in case author deletes it, overrides, new API breaks compatibility etc...

Reply:
same for url above

Reply:
Please do not comment on closed and handled PR's, there is no way to properly track that.

Instead, please open up an issue. Thanks! 👍